core/monitor/monitor.go

```diff
@@ -92,49 +93,104 @@ func New(mclient pb.MutationServiceClient,
 	}, nil
 }
 
-// ProcessLoop continuously fetches mutations and processes them.
-func (m *Monitor) ProcessLoop(domainID string, period time.Duration) {
-	mutCli := mutationclient.New(m.mClient, period)
-	responses, errs := mutCli.StartPolling(domainID, 1)
-	for {
+// EpochPair is two adjacent epochs.
+type EpochPair struct {
+	EpochA, EpochB *pb.Epoch
```

Comment:
Could simply `A` and `B` suffice, since the type is `EpochPair` and the fields are `*pb.Epoch` ?

Reply:
Sounds good.